src/test/scala/com/cognite/sdk/scala/common/OAuth2ClientCredentialsTest.scala

```diff
@@ -141,20 +141,23 @@ class OAuth2ClientCredentialsTest extends AnyFlatSpec with Matchers with OptionV
       cdfProjectName = "irrelevant"
     )
 
-    val io: IO[Unit] = for {
+    val io = for {
       authProvider <- OAuth2.ClientCredentialsProvider[IO](credentials,
-        refreshSecondsBeforeExpiration = 1,
-        Some(TokenState("firstToken", Clock[IO].monotonic.unsafeRunSync().toSeconds + 4, "irrelevant")))
+        refreshSecondsBeforeExpiration = 2,
+        Some(TokenState("firstToken", Clock[IO].realTime.map(_.toSeconds).unsafeRunSync() + 4, "irrelevant")))
       _ <- List.fill(5)(authProvider.getAuth).parUnorderedSequence
-      _ <- IO(numTokenRequests shouldBe 0) // original token is still valid
+      first <- numTokenRequests.get  // original token is still valid
```

Comment:
I think `_ <- numTokenRequests.get.map(numRequests => numRequests should be 0)` (or `_ <- numTokenRequests.get.map(n => n should be 0)` or maybe even `_ <- numTokenRequests.get.map(_ should be 0)` is better, more similar to what we used to have and slightly better in the sense that it will correctly abort early if an assertion fails.

Reply:
updated, thanks hehe 👍